coming-soon.html

```diff
@@ -0,0 +1,170 @@
+                    </g>
+                </g>
+            </svg>
+            </div>
+
+        </div>
+    </div>
+
+<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script>
+<script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='BIRTHDAY';ftypes[3]='birthday';}(jQuery));var $mcj = jQuery.noConflict(true);
+</script>
```

Comment:
What is this minified magic here? 🦄 

Is this the official way to do it? Did you copied it from an article somewhere? Basically, give me some intel, otherwise, I don't know